index.html

```diff
@@ -28,5 +29,93 @@ <h1>I’m Amine Glade to see you!</h1>
         <li><a href="#"><img src="/Icons/medium.png" alt="medium"></a></li>
       </ul>
   </section>
+  <!-- Begin The ToolBar and the Headline-->
+
+  <!-- Begin Work section-->
+  <section class="Work">
+    <div class="item Work-1">
+      <img src="/Work-Profolio/1.png" alt="Tonic">
+      <h1>Tonic</h1>
+      <ul class="List">
+        <li class="List-1st">CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="Work-List">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <h3>See Project</h3>
```

Comment:
- [ ] The button (e.g., "See project") should be a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed. Kindly work on that.